src/test/forms/formHelpers.test.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { ManagedNumberField } from '../../forms/formHelpers';
+
+test('ManagedNumberField should match snapshot', () => {
+  const wrapper = shallow(
+    <ManagedNumberField
+      format={ jest.fn() }
+      value={ 0 } />
+  );
+  expect(wrapper).toMatchSnapshot();
+});
+
+test('should set focused state to true on focus', () => {
+  const value = 10;
+  const wrapper = shallow(
+    <ManagedNumberField
+      format={ jest.fn() }
+      value={ value } />
+  );
+  wrapper.find('FormInput').simulate('focus', { target: { value }}, undefined);
```

Comment:
Any reason to have `undefined` as the third argument? It looks like Enzyme's `simulate` only normally takes an event name and mock event object.